lib/cli/Cargo.toml

```diff
@@ -54,6 +54,7 @@ cfg-if = "1.0"
 fern = { version = "0.6", features = ["colored"], optional = true }
 log = { version = "0.4", optional = true }
 tempfile = "3"
+unix_mode = "0.1.3"
```

Comment:
This should be required only in unix systems

Reply:
Good point, the entire module is `#[cfg(target_os = 'linux')]`. Fixed.

Reply:
Indeed, the entire module is #[cfg(target_os = 'linux')]. Ty, fixed.